testutils/channel_opts.go

```diff
@@ -124,10 +127,16 @@ func (o *ChannelOpts) DisableLogVerification() *ChannelOpts {
 
 // AddLogFilter sets an allowed filter for warning/error logs and sets
 // the maximum number of times that log can occur.
-func (o *ChannelOpts) AddLogFilter(filter string, maxCount uint) *ChannelOpts {
+func (o *ChannelOpts) AddLogFilter(filter string, maxCount uint, fields ...string) *ChannelOpts {
```

Comment:
I don't love the implicit requirement for an even number of fields, but taking `LogFields` would be really verbose.